resources/json/skiplaggedCoords.json

```diff
@@ -0,0 +1,166 @@
+[
+   "20,-160",
```

Comment:
As of now the files have the same content, but just because I had no time to run an initial "global" scan. So they will stay separated.